### PR TITLE
feat: implementation of registration process

### PR DESCRIPTION
**Pull request by j0nathansf**

### CODE REVIEW

File: src/components/Text/index.tsx [hunk 1]
```diff
@@ -15,21 +14,20 @@ export interface TextProps {
...
+  color: ${(props: { color: TextProps['color']; theme: ThemeAttributes }) =>
+    props.color && `${props.theme[props.color]}`};
   font-size: ${(props: { size: string }) => `${props.size}px`};
   font-family: ${(props: { font: string }) => `${props.font}`};
   max-width: ${(props: { maxWidth: number }) =>
     props.maxWidth ? `${props.maxWidth}px` : '100%'};
   text-align: ${(props: { textAlign: string }) => `${props.textAlign}`};
-  text-decoration: ${(props: { textDecoration: string }) => `${props.textDecoration}`};;
+  text-decoration: ${(props: { textDecoration: string }) =>
+    `${props.textDecoration}`}; ;
```
Comment (cindy-sf) [inline src/components/Text/index.tsx]: Warning, there are two semicolons

File: src/views/AccountCreation/index.tsx [hunk 2]
```diff
@@ -24,8 +25,66 @@ interface Props {
   navigation: NavigationContainerRef
 }
 
+interface UserData {
+  mail: string
+  tag: string
+  name: string
```
Comment (cindy-sf) [inline src/views/AccountCreation/index.tsx:31]: It is just a sugestion, but maybe you could put user picture in the same state ?

**Review by cindy-sf — DISMISSED** [review]

File: src/components/Text/index.tsx (hunk 1 above)
Comment (j0nathansf) [reply src/components/Text/index.tsx]: It happened when I rebase main
I fix it right away

File: src/views/AccountCreation/index.tsx (hunk 2 above)
Comment (j0nathansf) [reply src/views/AccountCreation/index.tsx:31]: Good idea 😀

**Review by cindy-sf — APPROVED** [review]

**Pull request merged**